build/Makefile

```diff
@@ -501,7 +505,11 @@ testrun : test
 TEST_STATUS:
 	@echo "========================= Performing Building provider tests"
 
-test : TEST_STATUS $(SCXPAL_INTERMEDIATE_DIR) $(INTERMEDIATE_DIR)/testrunner
+TEST_DATA_PREP:
+	@echo "========================= Performing Test data preparation"
+	cp -R $(TEST_DIR)/omi_preexec/testdata/cnfs $(INTERMEDIATE_DIR)/.
```

Comment:
The idea here is that we want to make sure the vendor files, as shipped, work properly. Currently, they do not, and we'd like to make sure that they do, preferably without modifications.

Also, I would expect that, over time as MySQL itself is modified and more versions come out, we'll check in more vendor files (and associated unit tests). That's exactly what happened with GetLinuxOS, anyway.

---
In reply to: [67194969](https://github.com/Microsoft/MySQL-Provider/pull/5#discussion_r67194969) [](ancestors = 67194969)